1-exercises/A-array-find/exercise.js

```diff
@@ -18,6 +18,10 @@ let names = [
 ];
 
 let longNameThatStartsWithA = findLongNameThatStartsWithA(names);
+function findLongNameThatStartsWithA (names) {
+  let longName = names.find(names => names.length>7 && names.startsWith("A"));
+  return longName;
+}
```

Comment:
There's a simpler way to find words that starts with "A". Instead of startWith method, can you think of accessing first letter using array[index]?